Group owner can self restore the group (#787)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/create-groups/restore-deleted-group.md

---

# Restore a deleted Office 365 Group

> The owner of an Office 365 group can restore the group by self.

Reviewer: @Batre-MSFT - Please merge this commit as well. Thanks.

> Steps:

Reviewer 2: ```suggestion
```

1) Access the deleted groups page by logging in to this link: https://outlook.office.com/people/group/deleted 
or 

> Login to Outlook for Web (OWA), click on Manage groups option, present under the Groups node, and then click on "Deleted" option

Reviewer 2: ```suggestion
log in to Outlook for Web (OWA), select the **Manage groups** option under the **Groups** node, and then choose **Deleted**.
```

2) The Office 365 groups deleted by the owner will be displayed on the right hand pane. 

> 3) From here, the group owner can click on "Restore" and the group will be restored

Reviewer 2: ```suggestion
```

> If the deleted group doesn't show here, the owner should contact person with administrative permissions.

Reviewer 2: ```suggestion
If the deleted group doesn't appear here, the group owner should contact a person with administrative permissions.
```
